### PR TITLE
Feature/certificados documentos

### PR DESCRIPTION
**Pull request by clmtest**
# FooCamp - Escuela de Robotica

## Trello ticket
[https://trello.com/c/EoTT8pt7](link)

## Description
Development of the new documents component 

## To reproduce

1. Run the project - npm run dev
2. Open the specific page (http://localhost:8080/christian-page.html)
3. Check the component 

## Screenshots
Documents

### CODE REVIEW

File: templates/document.hbs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<li class="document">
+    <img class="document__icon" src="../icons/download.svg" alt="">
```
[the remark this answers is not in the document]
Comment (clmtest) [reply templates/document.hbs]: Done

File: templates/documents.hbs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<div class="documents component">
+  <ul>
```
Comment (diegoamt) [inline templates/documents.hbs:2]: The title of the list is missing from Contentful. Please add the title as look in the design

**Review by diegoamt — CHANGES_REQUESTED** [review]

**Review by camilomejiag — APPROVED** [review]

File: templates/documents.hbs (hunk 2 above)
Comment (clmtest) [reply templates/documents.hbs:3]: done

File: templates/documents.hbs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<div class="documents component">
+  <h2 class="document__title">DOCUMENTOS</h2>
```
Comment (diegoamt) [inline templates/documents.hbs]: please use {{ title }} instead of DOCUMENTOS

**Review by diegoamt — CHANGES_REQUESTED** [review]

File: templates/documents.hbs (hunk 3 above)
Comment (sofiaperezm) [reply templates/documents.hbs]: done :)

**Pull request merged**